wslpy/exec.py

```diff
@@ -3,6 +3,42 @@
 This is the execution class to execute commands from different windows executables
 """
 import subprocess
+import os
+from wslpy.__init__ import isWSL
+import sys
+
+
+def __isInteropEnabled__():
+    cat_wslInterop = subprocess.getoutput(
+        "cat /proc/sys/fs/binfmt_misc/WSLInterop | grep '.*abled'")
+    return cat_wslInterop
+
+
+def isInteropEnabled():
+    """
+    Checks if interoperablility is enabled.
+
+    Returns
+    _______
+    A boolean value, `True` if interoperablility is enabled.
+    """
+    if __isInteropEnabled__() == 'enabled':
+        return True
+    else:
+        return False

```

Comment:
just `return __isInteropEnabled__() == 'enabled'` is good enough

Reply:
Copy that